src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public enum Color {
+    RED("Red"),
+    ORANGE("Orange"),
+    YELLOW("Yellow"),
+    GREEN("Green"),
+    BLUE("Blue"),
+    PURPLE("Purple"),
+    WHITE("White"),
+    BLACK("Black");
+    private final String value;
+
+    Color(String value) {
+        this.value = value;
+    }
+
+    public String getValue() {
+        return value;
+    }
```

Comment:
Good point. But you either can use `Color.name().toLowerCase()`, see next comment.